### PR TITLE
Add landing page for component library

### PR DESCRIPTION
**Pull request by suryaraj1**

### CODE REVIEW

File: blazeUI_css/main.css [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+}
+
+.hero-logo-container .hero-logo {
+  font-size: 94px;
+  font-weight: 400;
+}
+
+.hero-logo-container .hero-logo span {
+  font-weight: 600;
+  background: rgb(2, 0, 36);
```
Comment (Logan1x) [inline blazeUI_css/main.css:90]: Could have used color variables here also.

File: blazeUI_css/main.css [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    rgba(0, 86, 208, 1) 35%,
+    rgba(0, 212, 255, 1) 100%
+  );
+  background-clip: text;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.hero-subtext-container .hero-subtext {
+  color: #a0a0a0;
```
Comment (Logan1x) [inline blazeUI_css/main.css:103]: same here

**Review by Logan1x — APPROVED** [review]
Everything else looks good to me :)
Suggestion. You can connect to netlify or vercel, so that in the future if you push any pr/commit, it will show you a deployment previews.

File: blazeUI_css/main.css (hunk 2 above)
Comment (suryaraj1) [reply blazeUI_css/main.css:103]: Sure, fixing these.

File: blazeUI_intro/main.css [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  background: var(--blaze-color-blue-darker);
+}
+
+.btn-outline {
+  /* can remove this alignment fix later */
+  position: relative;
+  right: 20px;
+  transition: 0.3s ease;
+  color: var(--blaze-color-blue);
+  background: #000000;
```
Comment (Rohanmond) [inline blazeUI_intro/main.css:145]: Everything looks good ..only u can use the color variables instead of hard coded colors

**Review by Rohanmond — APPROVED** [review]

File: blazeUI_intro/main.css (hunk 3 above)
Comment (suryaraj1) [reply blazeUI_intro/main.css:145]: Fixing it.

**Pull request merged**